internal/allocator/allocator_test.go

```diff
@@ -482,29 +738,10 @@ func assignedPool(a *Allocator, svc string) string {
 	return a.svcToPool[svc]
 }
 
-func pools(pools ...map[string]*config.Pool) map[string]*config.Pool {
-	ret := map[string]*config.Pool{}
-	for _, p := range pools {
-		for k, v := range p {
-			ret[k] = v
-		}
-	}
-	return ret
-}
-
-func pool(name string, avoidBuggyIPs, autoAssign bool, cidrs ...string) map[string]*config.Pool {
-	ret := &config.Pool{
-		AvoidBuggyIPs: avoidBuggyIPs,
-		AutoAssign:    autoAssign,
-	}
-	for _, cidr := range cidrs {
-		_, n, err := net.ParseCIDR(cidr)
-		if err != nil {
-			panic(fmt.Sprintf("malformed CIDR %q", cidr))
-		}
-		ret.CIDR = append(ret.CIDR, n)
-	}
-	return map[string]*config.Pool{
-		name: ret,
+func ipnet(s string) *net.IPNet {
+	_, n, err := net.ParseCIDR(s)
+	if err != nil {
+		panic(err)
```

Comment:
`panic`? Either we will never hit this, or otherwise just return the error and deal with it?

Reply:
If we hit this, it means the test source code has an invalid IP network in it. It should never trigger when testing the code, it's just a shortcut to construct a *net.IPNet.